packages/transformers/svg-react/src/SvgReactTransformer.js

```diff
@@ -3,41 +3,61 @@
 import {Transformer} from '@parcel/plugin';
 
 import path from 'path';
-import camelcase from 'camelcase';
 import svgoPlugin from '@svgr/plugin-svgo';
 import jsxPlugin from '@svgr/plugin-jsx';
 import {transform} from '@svgr/core';
 
-function getComponentName(filePath) {
-  let validCharacters = /[^a-zA-Z0-9_-]/g;
-  let name = path.parse(filePath).name.replace(validCharacters, '');
-  return camelcase(name, {
-    pascalCase: true,
-  });
-}
-
 export default (new Transformer({
   async loadConfig({config}) {
-    let conf = await config.getConfig(['.svgrrc.json', '.svgrrc']);
-    return conf?.contents;
+    let svgrResult = await config.getConfig([
+      '.svgrrc',
+      '.svgrrc.json',
+      '.svgrrc.js',
+      '.svgrrc.cjs',
+      'svgr.config.json',
+      'svgr.config.js',
+      'svgr.config.cjs',
+    ]);
+    let svgoResult = await config.getConfig([
+      '.svgorc',
+      '.svgorc.json',
+      '.svgorc.js',
+      '.svgorc.cjs',
+      'svgo.config.json',
+      'svgo.config.js',
+      'svgo.config.cjs',
+    ]);
```

Comment:
Looks like the SVG optimizer only supports `svgo.config.*`. Maybe we should match that here?